new-site/src/docs/elements/components/card/code.mdx

```diff
@@ -99,7 +118,10 @@ import Playground from '../../../../components/Playground';
   <div class="hds-card">
```

Comment:
This has also the same rendering issues mentioned before.

Reply:
It seems to work now 🤔